src/docs/sdk/trace-context.mdx

```diff
@@ -0,0 +1,228 @@
+---
+title: "Trace Contexts (Experimental)"
```

Comment:
This name is very unfortunate because SDKs already have a "Trace Context", an object that is sent in events (both errors and transactions) as `event.contexts.trace`.

Reply:
Also for @RaduW -- suggestions on how we name the new trace context, instead? How about "trace state" borrowing from W3C?

Reply:
Trace state works for me